File: backend/tactic/domain/contest/schemas/contest_participate_message.py
from typing import Optional


class ContestParticipateMessage:
    message: Optional[str] = None
    memberId: Optional[int] = None
    contestId: Optional[int] = None
    ticketCnt: Optional[int] = None

    def to_dict(self):
        return {
            "message": self.message,
            "memberId": self.memberId,
            "contestId": self.contestId,
            "ticketCnt": self.ticketCnt
        }

File: backend/member/src/main/java/com/olock/blockstock/member/domain/award/dto/ContestParticipateMessage.java
package com.olock.blockstock.member.domain.award.dto;

import lombok.Getter;

import java.util.List;

@Getter
public class ContestParticipateMessage {
    private String message;
    private Long memberId;
    private Long contestId;
    private Integer ticketCnt;
}
